css-positioning/relative.css

```diff
@@ -0,0 +1,44 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
+
+  /* your code here */
+  position: relative;
+  right: 200px;
+}
+
+#bulbasaur {
+  /* your code here */
+  position: relative;
+  top: 20px;
+}
+
+#charmander {
+  /* your code here */
+
+  position: relative;
+  top: 35px;
+}
+
+#squirtle {
+  /* your code here */
+  position: relative;
+  top: 80px;
+}
+
+#pikachu {
+  /* your code here */
+  position: relative;
+  top: 160px;
+}
+
+#jigglypuff {
+  /* your code here */
+  position: relative;
+  top: 300px;
+}
```

Comment:
Instead of setting `position: relative` on each rule individually, you should set it once on the `.pokemon > img` rule above, and remove it from these